lib/src/subnav/subnav.component.scss

```diff
@@ -0,0 +1,30 @@
+@import "../colors.scss";
+
+$subnav-z-index: 1020 !default;
+
+.subnav {
+    width: 100%;
+    min-height: 52px;
+    box-sizing: border-box;
+    margin-bottom: 22px;
+    padding: 8px 20px;
+    background-color: $slate-gray-200;
+    border-bottom: 1px solid $slate-gray-300;
+    display: flex;
+    align-items: center;
+    z-index: $subnav-z-index;
+
+    p {
+        margin-bottom: 0;
+    }
+}
+
+.subnav::ng-deep .subnav-right {
```

Comment:
::ng-deep is probably ok for now, but I saw @corytak mention elsewhere that it is depreciated, and we may want to find ways to achieve the styles we want without it